src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -138,6 +138,28 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
             ValueFromPipelineByPropertyName = true)]
         public string[] Zone { get; set; }
 
+        [Parameter(
```

Comment:
Also add the edit to the Compute Changelog.md file.

Reply:
Yes, this is a draft PR I'll add help docs and update Change log files in the actual PR